lib/survey_gizmo/resource.rb

```diff
@@ -35,14 +36,27 @@ def all(conditions = {}, filters = nil)
         # SurveyGizmo claims they will fix this bug and eventually all questions will be
         # returned in one request.
         if self == SurveyGizmo::API::Question
-          _collection += _collection.map { |question| question.sub_questions }.flatten
+          collection += collection.map { |question| question.sub_questions }.flatten
         end
 
-        _collection
+        collection
+      end
+
+      # Do the pagination for you
+      def everything(conditions = {}, filters = {})
+        page = 1
+        collection = []
+
+        while !(request_collection = all(conditions, filters.merge(page: page))).empty?
```

Comment:
actually it's probably slicker to look at the `pages` part of the response than hit them with an extra web request just to make sure you got all the pages, so maybe hold off on merging this.  feedback still welcome.